client/src/datafiles/layouts/FileListingLayout.tsx

```diff
@@ -42,8 +42,13 @@ export const FileListingLayout: React.FC = () => {
     'designsafe.storage.frontera.work',
   ].includes(system);
 
+  const isUserWorkSystem = ['cloud.data'].includes(system);
```

Comment:
This is defined twice, should we have one import from the other?

Reply:
Since it's a 1-line array lookup I'd say "ehh". I think it makes more sense to define the work/home systems in a consants.ts somewhere

Reply:
I updated it so that USER_MYDATA_SYSTEM and USER_WORK_SYSTEM are exported from the hooks module.